GameFighter/Models/Warrior.cs

```diff
@@ -0,0 +1,18 @@
+﻿namespace GameFighter
+{
+    public class Warrior
+    {
+        public int Health { get; private set; }
+        public int Attack { get; protected set; }
+        public bool Alive { get => Health > 0 ? true : false; }
+        public Warrior()
+        {
+            Health = 50;
+            Attack = 5;
+        }
+        public void Hit(int attack)
```

Comment:
For me 'hit' means that the action is directed at someone else. In your case this means that this object is hit (passive scenario). I would rename this method or change the logic to reflect what the method actually does.